extensions/notebook/src/jupyter/serverInstance.ts

```diff
@@ -99,7 +99,7 @@ export class ServerInstanceUtils {
 	}
 }
 
-export class PerNotebookServerInstance implements IServerInstance {
+export class PerDriveServerInstance implements IServerInstance {
```

Comment:
It seems kind of weird to have this be the name when there's really nothing making this actually only be a single one per drive except for how it happens to be used

Reply:
Changed to PerFolderServerInstance. If you have better ideas, please let me know